tests/Console/Output/OutputContextTest.php

```diff
@@ -15,9 +15,8 @@
 namespace PhpCsFixer\Tests\Console\Output;
 
 use PhpCsFixer\Console\Output\OutputContext;
-use PHPUnit\Framework\TestCase;
+use PhpCsFixer\Tests\TestCase;
```

Comment:
*observation*

so nice commits history and then boom, this change, totally not related to commit log ;P